modules/guardduty/variables.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_version = "~> 0.11.0"
+  backend          "local"          {}
+}
+
+provider "aws" {
+  region  = "${var.region}"
+  version = "~> 2.5"
+}
+
+variable "region" {
+  default = "eu-west-1"
+}
+
+variable "publish_frequency" {
+  default = "FIFTEEN_MINUTES"
+}
+
+variable "landing_account_id" {}
```

Comment:
Is it that this module can only enable guard duty on the landing account?  Would we not want guard duty enabled on all our accounts?

Reply:
I'm indeed planning to enable it in all AWS accounts we have
To start with, I'm just enabling it in the landing account, I would then re-iterate for deploying it in different accounts:
- Landing account would send GuardDuty invitations to member accounts
- Member accounts enable and accept GuardDuty invitations
We would then make sure that all GuardDuty events are gather in 1 account only